AppCenter/AppCenter/MSChannelProtocol.h

```diff
@@ -1,9 +1,6 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-#ifndef MS_CHANNEL_PROTOCOL_H
-#define MS_CHANNEL_PROTOCOL_H
```

Comment:
What was this used for? I feel like I am discovering this now. It looks like a hack. I wonder if that was at the time of the event filter service.

Reply:
it was for carthage build to deal with core dependency conflicts in modules. It's not a root cause and the wrong way to solve the problem. It isn't required now because I've cleaned up the dependencies setup.
Even when/if we move to proper framework references during modules build, it shouldn't be there